<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRSPlannerWindow</class>
 <widget class="QMainWindow" name="CRSPlannerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Robot-Airship Facility  -  Motion Planner</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Positions"/>
    <addaction name="actionSave_Positions"/>
    <addaction name="actionLoad_Planner"/>
    <addaction name="actionSave_Planner"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="config_dock">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_Positions">
   <property name="text">
    <string>Load Positions...</string>
   </property>
  </action>
  <action name="actionSave_Positions">
   <property name="text">
    <string>Save Positions...</string>
   </property>
  </action>
  <action name="actionLoad_Planner">
   <property name="text">
    <string>Load Planner...</string>
   </property>
  </action>
  <action name="actionSave_Planner">
   <property name="text">
    <string>Save Planner...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
